torch/nn/attention/_flex_attention.py

```diff
@@ -42,7 +42,7 @@ def _identity(
 _DEFAULT_SPARSE_BLOCK_SIZE = 128
 
 
-class _BlockSparseMask:
+class _BlockMask:
```

Comment:
I still keep this class definition to keep record of the structure of a block mask, since tuple doesn't have explicit field names.